packages/core/android/src/main/java/io/sentry/react/RNSentryMapConverter.java

```diff
@@ -28,21 +30,85 @@ private RNSentryMapConverter() {
     throw new AssertionError("Utility class should not be instantiated");
   }
 
-  public static Object convertToWritable(@Nullable Object serialized) {
+  public interface WritableArrayCreator {
+    WritableArray create();
+  }
+
+  public interface WritableMapCreator {
+    WritableMap create();
+  }
+
+  public static class WritableJavaArrayCreator implements WritableArrayCreator {
+    @Override
```

Comment:
Is the annotation needed? We don't override already implemented method here.

Reply:
Technically it is not needed but it is usually recommended ([example style guide](https://google.github.io/styleguide/javaguide.html#s6.1-override-annotation)) to ensure that the method correctly matches the method signature in the interface.